int32 tavolo
float32 x
float32 y
